### ISSUE TITLE
[Suggestion] Thermite blend & Crucible

### ISSUE THREAD

**Issue by jburrows82** [TeamBR-Modding/NeoTech#372]
You're looking for more end game stuff and wanting to mess with new alloys -- how about starting with an early game mechanic like a Blast/coke oven that acts as a crucible (not a furnace), but requires a thermite reaction to start and reach high enough heat to work. In this crucible block (or multi-block?) you produce Steel/Hardened Iron/Enriched Iron/whatever alloys, and even Lava (20mb per tick, one bucket per 50 seconds?). The heat drops off quicker than a normal furnace and burns (carbon) fuels quicker than normal furnaces. If heat drops below 70% it begins to super-cool at triple the rate and another exothermic reaction is required. I don't know what would work for balance purposes on fuel efficiency, but I'm thinking half of whatever vanilla furnaces are? The crucible is the basic non-rf entry into alloy smelting (like a Smeltery, but with higher upkeep) before getting an RF Alloy Smelter. 

How do you cause a thermite reaction? Igniting thermite powder with a flint and steel of course! Thermite is blended using Gunpowder, Iron dust, Coal/Charcoal, and Flint. Can also be used to make throwable Mining Dynamite (Redstone Torch, Thermite, Button). 

Basically I want an early game Lava generation. Putting a torch under a crucible is too cheap and easy. But a crucible idea could easily work with alloys, too. And throwable TNT? Yea, you know that would be fun.

**Comment by pauljoda** [comment]
I like the idea, but not really in line with current concept. Maybe in another mod or something like Assisted Progression
